### PR TITLE
(WIP) docs: update the tech guide for video.js 6

### PR DESCRIPTION
**Pull request by BrandonOCasey**
Still needs some work particularly in the creation examples

### CODE REVIEW

File: docs/guides/tech.md [hunk 1]
```diff
@@ -1,120 +1,88 @@
...
 
-canPlayType
-play
-pause
-currentTime
-volume
-duration
-buffered
-supportsFullScreen
+When writing a Tech you will have to integrate your technologies API with the existing API. In the following examples
```
Comment (misteroneill) [inline docs/guides/tech.md]: `technologies` should be `technology's`

File: docs/guides/tech.md [hunk 2]
```diff
@@ -1,120 +1,88 @@
 # Playback Technology ("Tech")
 
-Playback Technology refers to the specific browser or plugin technology used to play the video or audio. When using HTML5, the playback technology is the video or audio element. When using Flash, the playback technology is the specific Flash player used, e.g. Flowplayer, YouTube Player, video-js.swf, etc. (not just "Flash"). This could also include Silverlight, Quicktime, or any other plugin that will play back video in the browser, as long as there is an API wrapper written for it.
+Playback Technology refers to the specific browser or plugin technology used to play the video or audio.
+When using HTML5, the playback technology is the video or audio element. When using Flash and the
+[videojs-flash project](http://github.com/videojs/videojs-flash), the playback technology is the specific Flash player used,
+e.g. Flowplayer, YouTube Player, video-js.swf, etc. (not just "Flash"). This could also include Silverlight,
```
Comment (mister-ben) [inline docs/guides/tech.md]: When using Flash and videojs-flash the tech is video-js.swf.

The You Tube tech isn't a good example of a Flash-based tech any more. And Flowplayer?

Comment (BrandonOCasey) [inline docs/guides/tech.md]: Good call. This is all a carry over from the old doc. Let me fix it up some more.

File: docs/guides/tech.md [hunk 3]
```diff
@@ -1,120 +1,88 @@
 # Playback Technology ("Tech")
 
-Playback Technology refers to the specific browser or plugin technology used to play the video or audio. When using HTML5, the playback technology is the video or audio element. When using Flash, the playback technology is the specific Flash player used, e.g. Flowplayer, YouTube Player, video-js.swf, etc. (not just "Flash"). This could also include Silverlight, Quicktime, or any other plugin that will play back video in the browser, as long as there is an API wrapper written for it.
+Playback Technology refers to the specific browser or plugin technology used to play the video or audio.
+When using HTML5, the playback technology is the video or audio element. When using Flash and the
+[videojs-flash project](http://github.com/videojs/videojs-flash) (which was builtin to Video.js prior to Video.js 6), the playback technology is the specific Flash player used, and in most cases that will be [video-js-swf](https://github.com/videojs/video-js-swf). This could also include Silverlight,
```
Comment (mister-ben) [inline docs/guides/tech.md]: Should be "built into". "Video.js prior to version 6" would be a little better to avoid the tautology.

**Pull request closed without merging**